### PR TITLE
make possible to redefinition of build.gradle.in by user

### LINKED ISSUE #40: Add a way for the user to provide a custom gradle.build.in
Since different version of qt use different version of gradle in the `src/` folder, would it be possible to use our own `gradle.build.in` script.

* Qt5.13: `'com.android.tools.build:gradle:3.2.0'`
* Qt5.14: `'com.android.tools.build:gradle:3.5.0'`
* Qt5.15: `'com.android.tools.build:gradle:3.6.0'`

I tried just to update the script to 3.6 but build break with Qt 5.13.

Would you consider a PR that allow user to provide a `build.gradle.in` in the same way as `AndroidManifest.xml.in` ?

### PR DESCRIPTION
**Pull request by Dushistov**
fixes #40

### CODE REVIEW

File: AddQtAndroidApk.cmake [hunk 1]
```diff
@@ -132,6 +133,10 @@ macro(add_qt_android_apk TARGET SOURCE_TARGET)
             # custom manifest template provided
             set(QT_ANDROID_MANIFEST_TEMPLATE "${ARG_PACKAGE_SOURCES}/AndroidManifest.xml.in")
         endif()
+
+        if (EXISTS "${ARG_PACKAGE_SOURCES}/build.gradle.in")
+          set(BUILD_GRADLE_TEMPLATE "${ARG_PACKAGE_SOURCES}/build.gradle.in")
```
Comment (LaurentGomila) [inline AddQtAndroidApk.cmake]: Not correctly indented here (should be 4 spaces).

**Review by LaurentGomila — CHANGES_REQUESTED** [review]
Thanks for contributing.

Can you add this to the documentation please (readme.md)?

File: AddQtAndroidApk.cmake (hunk 1 above)
Comment (Dushistov) [reply AddQtAndroidApk.cmake]: fixed

**Comment by Dushistov** [comment]
> Can you add this to the documentation please (readme.md)?

I somehow missed your comment. I added mention of build.gradle.in into readme.

**Comment by kafeg** [comment]
Could this be merged please?

**Review by LaurentGomila — APPROVED** [review]

**Comment by LaurentGomila** [comment]
Sorry, I completely forgot about it.

**Pull request merged**